scheduler/src/cook/tools.clj

```diff
@@ -1030,6 +1030,15 @@
                   (str %))
                resource-map))
 
+(defn format-resource-map-for-structured-logging
+  "Given a map with resource amount values,
+   formats the amount values for structured logging"
+  [resource-map]
+  (pc/map-vals #(if (number? %)
```

Comment:
Of note: the `format-resource-map` function specifically checks fr floats. I am removing string formatting for all numbers.